Formula/darglint.rb

```diff
@@ -0,0 +1,26 @@
+class Darglint < Formula
+  include Language::Python::Virtualenv
+
+  desc "Python docstring argument linter"
+  homepage "https://github.com/terrencepreilly/darglint"
+  url "https://github.com/terrencepreilly/darglint/archive/refs/tags/v1.8.0.tar.gz"
```

Comment:
Can we use the source tarball from [PyPI](https://pypi.org/project/darglint/#files)? That would make tracking resources easier (automatic with `bump-formula-pr`/`update-python-resources`).

Reply:
Done! Ready for another review